src/app/(app)/contact/page.tsx

```diff
@@ -0,0 +1,164 @@
+                                                   className="block text-sm font-medium text-zinc-300 mb-2">
+                                                E-mail:
+                                            </label>
+                                            <Input
+                                                id="email"
+                                                type="email"
+                                                placeholder="Adja meg e-mail címét"
+                                                className="bg-zinc-900 border-zinc-700 text-zinc-100 pl-2"
+                                            />
+                                        </div>
+
+                                        <div>
+                                            <label htmlFor="message"
+                                                   className="block text-sm font-medium text-zinc-300 mb-2">
+                                                Üzenet:
+                                            </label>
+                                            <Textarea
+                                                id="message"
+                                                placeholder="Írja be üzenetét"
+                                                className="bg-zinc-900 border-zinc-700 text-zinc-100 min-h-[100px] pl-2"
+                                                style={{resize: 'none'}}
+                                            />
+                                        </div>
+
+                                        <div>
+                                            <Button className="w-full bg-zinc-700 hover:bg-zinc-600 text-zinc-100 mt-6">
+                                                Küldés
+                                            </Button>
+                                        </div>
+                                    </form>
+                                </CardContent>
+                            </Card>
+                        </div>
+
+                        <div className="space-y-8 h-full order-4 lg:order-4">
+                            <div className="relative h-full">
+                                <iframe
+                                    src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2696.9015870927266!2d19.057!3d47.477!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4741dda3f096c761%3A0x7f8be5816ab76f58!2zQnVkYXBlc3QsIEJlcmNzw6lueWkgdS4gMjgsIDExMTc!5e0!3m2!1sen!2shu!4v1710284151776!5m2!1sen!2shu"
+                                    width="100%"
+                                    height="100%"
+                                    style={{border: 0}}
+                                    allowFullScreen
+                                    loading="lazy"
+                                    referrerPolicy="no-referrer-when-downgrade"
+                                    className=""
+                                />
```

Comment:
The map pin seems incorrect.

It should be this: https://maps.app.goo.gl/bhmBaitSa5hAVXu69